analysis_options.yaml

```diff
@@ -1,13 +1,240 @@
-include: package:pedantic/analysis_options.1.8.0.yaml
+# This is a copy (as of March 2021) of flutter/flutter's analysis_options file,
```

Comment:
Note that if you took this from the root of the flutter repo, it gets "extended" in its subdirectories:

* [flutter/packages](https://github.com/flutter/flutter/blob/master/packages/analysis_options.yaml)
* [flutter/packages/flutter](https://github.com/flutter/flutter/blob/master/packages/flutter/analysis_options.yaml)

(I'm not sure which exact flavor is being used when we get tested, but I think this file at least needs the `no_runtimeType_toString` linter rule enabled.

Reply:
Good call, added the two extra rules in a marked section..